lib/activerecord-mysql-unsigned/active_record/v4/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -4,42 +4,34 @@ module ActiveRecord
   module ConnectionAdapters
     class AbstractMysqlAdapter < AbstractAdapter
  
-      class ChangeColumnDefinition < Struct.new(:column, :type, :options) #:nodoc:
+      class ChangeColumnDefinition < Struct.new(:column, :name) #:nodoc:
       end
```

Comment:
This change is works well.
Because, defined name space is different from the original `ChangeColumnDefinition`.
`change_column_sql` and `rename_column_sql` are modified so that new `ChangeColumnDefinition` defined here will be used.
If the name is ambiguous, you may rename to like `MyChangeColumnDefinition`.
The cause of `TypeError` is because defined the same name class with mysql-awesome.